frappe/database/database.py

```diff
@@ -540,7 +562,7 @@ def get_singles_value(self, *args, **kwargs):
 		return self.get_single_value(*args, **kwargs)
 
 	def _get_values_from_table(self, fields, filters, doctype, as_dict, debug, order_by=None,
-								update=None, for_update=False, run=True):
+								update=None, for_update=False, run=True, **kwargs):
```

Comment:
Passing kwargs around everywhere...I'm sweating :')

Reply:
Oh...so this is passed for `build_conditions` the function in query.py